core/lib/spree/app_configuration.rb

```diff
@@ -213,6 +213,10 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Integer] Promotions to show per-page in the admin (default: +15+)
     preference :promotions_per_page, :integer, default: 15
 
+    # @!attribute [rw] disable_actionless_promotion_validation
+    #   @return [Boolean] Promotions should have actions associated before activation (default: +true+)
+    preference :actionless_promotion_inactive, :boolean, default: false
```

Comment:
How about making this more explicit by calling it `consider_actionless_promotions_active` instead?

Reply:
Done!

https://github.com/solidusio/solidus/pull/3749/commits/24f884cc9705c0c4987a82fbd0fa83357e21b43f